src/sentry/static/sentry/app/utils/escapeSearchQuery.jsx

```diff
@@ -0,0 +1,11 @@
+/**
+ * Escapes search strings for use in events/issues search
+ */
+export function escapeSearchQuery(value) {
+  // Cast numbers to string
+  if (typeof value === 'number') {
+    return `${value}`;
+  }
+
+  return typeof value === 'string' ? value.replace(/"/g, '"') : '';
```

Comment:
What does the `replace` part of this actually do?

Reply:
Oops, I had `'\"'` which is wrong since `\` is an escape character (and prettier formatted it away).